javascript/chronometer.js

```diff
@@ -1,34 +1,41 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(callback) {
-    // ... your code goes here
+    callback ? callback() : false;
+    this.intervalId = setInterval(() => {
+      this.currentTime++;
+      console.log(this.currentTime);
+    }, 1000);
   }
```

Comment:
You are missing the callback function that could be passed:
```
this.intervalId = setInterval(() => {
   if (callback) {
callback()
}
    this.currentTime++
}, 1000) 
```
Yesterday you were told to run it outside the interval, it was a confusion with the instructions, we are sorry about that. Great job nonetheless!